scripts/run_mteb_english.py

```diff
@@ -23,6 +23,7 @@
     "MassiveScenarioClassification",
     "MTOPDomainClassification",
     "MTOPIntentClassification",
+    "NewsClassification",
```

Comment:
@Muennighoff do we typically extend the English benchmark with new tasks?

Reply:
No, so this should be removed I think. I think it makes most sense to leave the original MTEB English datasets as is for comparison with older models.
But in the paper or in another paper at some point, we can ofc decide to introduce a new English average that is better in some ways and then deprecate the old one.

Reply:
Thanks for clearing! Removed .